### PR TITLE
Added Scraping Hacker news website script

### PR DESCRIPTION
**Pull request by GudlaArunKumar**
#CH20

Issue #92 
About PR: Created script for scraping the first 2 pages of  Hacker news website wherein user can read some cool tech news as articles which has Upvotes more than 100.

### CODE REVIEW

File: Web-Scraping/ScrappingHackerNewsWebsite.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+def create_custom_hackernews(mega_link,mega_votes):
+    hackerNews =[]
+    for index,item in enumerate(mega_link):
+        title = mega_link[index].getText()  #To get title of the story(news)
+        href = mega_link[index].get('href',None) # To get link of stroy(news).If no link is present, default is None
+        vote = mega_votes[index].select('.score') # points are stored inside class "score" of class subtext,if points/votes not available, then class score wont be present.
+        if len(vote): #To check if class "score" exists or not
+            points = int(vote[0].getText().replace(' points', ''))
+            if points > 100:  # To get votes/points more than 100
+                hackerNews.append({'title': title, 'link': href,'votes':points})
```
Comment (Neilblaze) [inline Web-Scraping/ScrappingHackerNewsWebsite.py]: Update it to this ⬇
`hackerNews.append({'title': title, 'link': href, 'votes': points})`

Follow good code practice - [here](https://gist.github.com/sloria/7001839)
Signed by - @Neilblaze

File: Web-Scraping/ScrappingHackerNewsWebsite.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
+'''
+Scraping the first 2 pages of Hacker news website which gives lot of Tech news(as a articles)
+which has upvotes more than 100.User can just click on story link to see the article.
+'''
+
+'''
+Program uses requests module to get web data from URL and BeautifulSoup module to parse the web data
+as HTML using html parser.
+Install requests and BeautifulSoup module before executing!
```
Comment (Neilblaze) [inline Web-Scraping/ScrappingHackerNewsWebsite.py]: LGTM :+1: 
Better add README (guide) along with this inside the folder same as your `<filename>` & you're good to go!

Signed by - @Neilblaze

**Review by Neilblaze — CHANGES_REQUESTED** [review]

File: Web-Scraping/ScrappingHackerNewsWebsite.py (hunk 1 above)
Comment (GudlaArunKumar) [reply Web-Scraping/ScrappingHackerNewsWebsite.py]: Sure okay

File: Web-Scraping/ScrappingHackerNewsWebsite.py (hunk 2 above)
Comment (GudlaArunKumar) [reply Web-Scraping/ScrappingHackerNewsWebsite.py]: Okay. I will add readme inside the folder

**Comment by GudlaArunKumar** [comment]
@Neilblaze I have made the changes.Please check now

**Comment by Neilblaze** [comment]
Good job @GudlaArunKumar :tada:

**Review by Neilblaze — APPROVED** [review]
LGTM! :rocket: 
@ankitdobhal merge status OK!

**Pull request merged**